### PR TITLE
Feature/sort

### PR DESCRIPTION
**Pull request by osirisguitar**

### CODE REVIEW

File: src/routes/Selection/Selection.tsx [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import React from "react";
-import { Selection as SelectionType } from "../../store/selection/types";
+import { useState } from "react";
```
Comment (izabellalarsson) [inline src/routes/Selection/Selection.tsx:2]: import React, {useState} from 'react is cleaner.

File: src/routes/Selection/index.tsx [hunk 2]
```diff
@@ -7,18 +7,22 @@ import {
...
   deleteSelection: typeof deleteSelection;
+  sortSelection: typeof sortSelection;
   checkPopulationRegistraion: typeof checkPopulationRegistraion;
 }
 
 const SelectionContainer: React.FC<SelectionContainerProps> = ({
   selection,
   getSelection,
   deleteSelection,
+  sortSelection,
```
Comment (izabellalarsson) [inline src/routes/Selection/index.tsx:25]: This is just passed down to the Selection component, you could just import it in the Selection file.

File: src/routes/Selection/Selection.tsx [hunk 3]
```diff
@@ -70,83 +85,131 @@ interface SelectionProps {
...
-      <EmptyState>Inga kontrakt matchade.</EmptyState>
-    )}
-  </>
-);
+      {selection.contracts && selection.contracts.length > 0 ? (
+        <List>
+          <ListHeader>
+            <HeaderColumn
+              as="button"
+              onClick={() => sortSelection("name", sortOptions, selection)}
```
Comment (izabellalarsson) [inline src/routes/Selection/Selection.tsx:142]: Here I would make a list out of the different newSortColumn names and do a map.

File: src/routes/Selection/Selection.tsx [hunk 4]
```diff
@@ -70,83 +85,131 @@ interface SelectionProps {
...
-    <BackButton to="/" />
-    <h1>{selection.name}</h1>
-    <Header>
-      <div>
-        <div>
-          <strong>Sökterm:</strong> {selection.selection_term}
-        </div>
+  sortSelection,
+}) => {
+  const [sortOptions] = useState({
```
Comment (izabellalarsson) [inline src/routes/Selection/Selection.tsx:97]: This don't need to be a state since we never update it.

Comment (osirisguitar) [reply src/routes/Selection/Selection.tsx:97]: Yes we do, in the sort function.

Comment (izabellalarsson) [reply src/routes/Selection/Selection.tsx:97]: Sorry didn't get a notification about this. But if/when you update it you should updating it by using setSortOption. And pass it down to the sort fn aswell.

**Pull request merged**